src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -797,8 +797,11 @@ static int DoubleForDeclaration(int x)
 True
 True
 True
+True
 4
 True
+True
```

Comment:
A couple of instrumentation points have been added because a single declaration statement that initializes two variables now has two instrumentation points. This is defensible